src/components/filterBox/FilterBox.tsx

```diff
@@ -100,7 +100,6 @@ export class FilterBox extends React.Component<IFilterBoxProps, any> {
           onBlur={() => this.handleOnBlur()}
           onFocus={(e: React.FocusEvent<any>) => { this.placeCursorAtEndOfInputValue(e); }}
           onKeyDown={(e) => this.handleOnKeyDown(e)}
-          value={this.props.filterText}
```

Comment:
This is a wanted change?

Reply:
it is, I know I solved a problem with that change but i don't remember what it was exactly. I'll let you know if it comes to mind.

Reply:
since the filterbox change is now debounced, it was causing problems